### PR TITLE
KIT-274

### PR DESCRIPTION
**Pull request by emericb**

### CODE REVIEW

File: src/main/java/com/templateproject/api/controller/CollectionController.java [hunk 1]
```diff
@@ -43,17 +45,16 @@ public CollectionController(CollectionRepository collectionRepository) {
...
     }
 
-    @DeleteMapping("/collections/{id}")
+    @DeleteMapping("/{id}")
     public @ResponseBody void deleteCollection(@PathVariable(value = "id") Long id) {
         collectionRepository.deleteById(id);
     }
 
-    @PostMapping("/collections/search")
+    @PostMapping("/search")
```
Comment (Benja-san) [inline src/main/java/com/templateproject/api/controller/CollectionController.java:57]: For future you could optimise the controller by removing this route and add an optional quey to your "/" routes in order to get allDatas containing the request body part

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -31,3 +31,6 @@ spring.jpa.show-sql=true
 # schema will be automatically updated accordingly to java entities found in
 # the project
 spring.jpa.hibernate.ddl-auto=update
+
+# Naming strategy
+server.servlet.contextPath=/api
```
Comment (Benja-san) [inline src/main/resources/application.properties:36]: did you put this line in your "-dev" file as well ?

**Review by Benja-san — APPROVED** [review]

File: src/main/resources/application.properties (hunk 2 above)
Comment (emericb) [reply src/main/resources/application.properties:36]: Yes!

**Review by Tylyfox — APPROVED** [review]
Big function for search ^^

**Pull request merged**